install/build_from_source.md

```diff
@@ -56,7 +56,42 @@ To have ROOT setup automatically at each login, that command can be appended to
 
 ## ROOT, Python and PyROOT
 
-> TODO: explain what build options are relevant, what they do, Python2/Python3, etc.
+ROOT is built with Python bindings (PyROOT) if the option `-Dpyroot=ON` is present (enabled by default). If the old version of PyROOT (pre-v6.22) wants to be built, the option `-Dpyroot_legacy` needs to be specified.
+
+Starting from version 6.22, PyROOT libraries are built by default with both Python3 and Python2 if a version of CMake >= 3.14 is used. For each Python version X.Y used to build PyROOT (e.g. 3.8, 2.7, etc.) the following libraries (containing the C++ extensions) will appear both in the build directory and in the installation directory:
+- `libROOTPythonizationsX_Y.so`
+- `libcppyX_Y.so`
+- `libcppyy_backendX_Y.so`
+
+The following pure Python packages will appear as well:
+- `ROOT`
+- `cppyy`
+- `cppyy_backend`
+- `JupyROOT`
+- `JsMVA`
+
+If no option is specified, PyROOT will be built for the most recent Python3 and Python2 versions that CMake can find. If only one version can be found, PyROOT will be built for only that version. Moreover, for a given Python installation to be considered, it must provide both the Python interpreter (binary) and the development package. To build PyROOT, it is thus suggested to verify that python-dev is present and [install it](https://pypi.org/project/python-dev-tools/) if not.
+
+In order to build with specific Python installations (not necessarily the highest ones) hints to CMake can be provided by using `-DPython2_ROOT_DIR=python2_dir` and/or `-DPython3_ROOT_DIR=python3_dir` to point to the root directory of some desired Python installation. Similarly, `Python2_EXECUTABLE` and/or `Python3_EXECUTABLE` can be used to point to particular Python executables.
+
+PyROOT can be built with only one version of Python even if multiple installations are present in the system. For this purpose, the option `-DPYTHON_EXECUTABLE=/path/to/python_exec` can be used to point to the desired Python installation.
```

Comment:
i don't understand how `-DPYTHON_EXECUTABLE` interacts with `Python2_EXECUTABLE` and `Python3_EXECUTABLE`, does it take precedence? if yes, please mention it in the text